integration_tests/pages/match/bookASpacePage.ts

```diff
@@ -1,13 +1,9 @@
-import { ApType, PlacementDates, PlacementRequestDetail, Premises } from '@approved-premises/api'
+import { ApType, PlacementCriteria, PlacementDates, PlacementRequestDetail, Premises } from '@approved-premises/api'
```

Comment:
I we want to be using `Cas1SpaceBookingCharacteristic` here -- there's a slight disconnect between `PlacementRequest` using `PlacementCriteria`, but from what I can see the types seem to have enough overlap that we should be able to migrate to the new one bit by bit. We may be able to update the API to start using those instead?

Reply:
For reference, the `criteria` query parameter on the occupancy view already uses the new type: https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/main/server/controllers/match/placementRequests/occupancyViewController.ts#L33